examples/elevenlabs-stt/README.md

```
@@ -0,0 +1,404 @@
+
+```bash
+# Verify your API key is set
+echo $ELEVENLABS_API_KEY
+
+# Or set it inline
+ELEVENLABS_API_KEY=your_key promptfoo eval
+```
+
+### Audio File Not Found
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)